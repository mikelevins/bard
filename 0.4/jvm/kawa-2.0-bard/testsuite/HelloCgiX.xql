declare boundary-space preserve;
response-header("X-Count", 6+7),
response-header("Content-Type", "text/html"),
unescaped-data("<?xml version='1.0'?>
"),
<body>
  <p>The translated path was: {request-path-translated()}</p>,
  <p>{let $query := qexo:request-query-string() return
      if ($query)
      then ("The query string was: ",$query)
      else "There was no query string."}</p>
</body>
