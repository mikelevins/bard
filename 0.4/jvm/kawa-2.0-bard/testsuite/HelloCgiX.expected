Status: 200
X-Count: 13
Content-Type: text/html

<?xml version='1.0'?>
<body>
  <p>The translated path was: ./HelloCgiX</p>,
  <p>There was no query string.</p>
</body>
